packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Models/Types/ModelTypeProviderTests.cs

```diff
@@ -176,19 +176,11 @@ public void BuildConstructor_ValidateConstructors()
             var ctors = modelTypeProvider.Constructors;
             Assert.IsNotNull(ctors);
 
-            Assert.AreEqual(3, ctors.Count);
+            Assert.AreEqual(1, ctors.Count);
```

Comment:
Can we add some tests for serialization ctors?

Reply:
The serialization constructor tests should be in [MrwSerializationTypeProviderTests.cs](https://github.com/microsoft/typespec/pull/3414/files#diff-3a04add317d9c4fd329e876dc9f69bdd165dbfedcbe94a02ea4f21af682a15af). Were you looking for a specific scenario to cover?